[2024-12-30 11:45:33,673] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag_V3.third_task manual__2024-12-30T11:45:32.877796+00:00 [queued]>
[2024-12-30 11:45:33,680] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag_V3.third_task manual__2024-12-30T11:45:32.877796+00:00 [queued]>
[2024-12-30 11:45:33,680] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-12-30 11:45:33,680] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2024-12-30 11:45:33,680] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-12-30 11:45:33,690] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): third_task> on 2024-12-30 11:45:32.877796+00:00
[2024-12-30 11:45:33,695] {standard_task_runner.py:52} INFO - Started process 11665 to run task
[2024-12-30 11:45:33,697] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag_V3', 'third_task', 'manual__2024-12-30T11:45:32.877796+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp94qznxml', '--error-file', '/tmp/tmpl81kvulj']
[2024-12-30 11:45:33,698] {standard_task_runner.py:80} INFO - Job 20: Subtask third_task
[2024-12-30 11:45:33,747] {task_command.py:370} INFO - Running <TaskInstance: first_dag_V3.third_task manual__2024-12-30T11:45:32.877796+00:00 [running]> on host 0766066e7c2f
[2024-12-30 11:45:33,825] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=swetansu
AIRFLOW_CTX_DAG_ID=first_dag_V3
AIRFLOW_CTX_TASK_ID=third_task
AIRFLOW_CTX_EXECUTION_DATE=2024-12-30T11:45:32.877796+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-30T11:45:32.877796+00:00
[2024-12-30 11:45:33,826] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-12-30 11:45:33,827] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo i am task3 and will be running after task1']
[2024-12-30 11:45:33,841] {subprocess.py:85} INFO - Output:
[2024-12-30 11:45:33,842] {subprocess.py:92} INFO - i am task3 and will be running after task1
[2024-12-30 11:45:33,842] {subprocess.py:96} INFO - Command exited with return code 0
[2024-12-30 11:45:33,875] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag_V3, task_id=third_task, execution_date=20241230T114532, start_date=20241230T114533, end_date=20241230T114533
[2024-12-30 11:45:33,911] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-30 11:45:33,935] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
